scripts/generate-erc20.js

```diff
@@ -149,7 +150,8 @@ const start = async () => {
     "^[a-zA-Z0-9+\\-%/\\$\\.]+$";
   const validate = ajv.compile(schema);
   if (validate(MaskTokenList)) {
-    process.stdout.write(JSON.stringify(MaskTokenList));
+    const riskCheckedList = await riskCheck(MaskTokenList);
+    process.stdout.write(JSON.stringify(riskCheckedList));
```

Comment:
Dump the risk data into the final token list is unnecessary. Instead write into an audit file that let the reviewer to check the file.

Reply:
modified